### DISCUSSION TITLE
Different connection to database for security

### DISCUSSION THREAD

**Discussion by Jul000** [laravel/nova-issues#3769] [Q&A]
Hello,

Is it possible to use nova with a different user (connection) for the database (with more privileges, like delete) than the one in the app (less privileges, without delete for example) ? Will it improve the security ? 

Thanks for your advices 

Julien

**Comment by crynobone** [comment]
If you set up authorization (check the documentation) properly I wouldn't see how having 2 different databases can make any security improvements.
